bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -48,12 +48,17 @@ spec:
             set -o allexport
             source "/env/${K8S_NODE}"
             set +o allexport
-          fi
+          fi  
+          
           echo "$(date -Iseconds) - starting ovn-controller"
           exec ovn-controller unix:/var/run/openvswitch/db.sock -vfile:off \
             --no-chdir --pidfile=/var/run/ovn/ovn-controller.pid \
+            --syslog-method="{{.OVNPolicyAuditDestination}}" \
```

Comment:
shouldn't these flags be passed only if audit logging is enabled?

Reply:
So based on the upstream work this feature is always enabled internally in ovn-k.... However it will only be activated if the user passes the required annotations on a namespace's yaml.

If the user does not manually configure the audit-logging-config all of the values will default to the values I have specified (in the case of the `--syslog-method` flag it will just default to the OVN-K default of `null`)

Therefore I think maintaining a single path with defined defaults is easier here.

Reply:
Is there any way to turn off audit logging? What happens in that case?

This all is confusing. If audit logging is disabled, I would expect to see no configuration for it.

Reply:
aha, I see now. audit logging is always enabled.